### DISCUSSION TITLE
Feature Request: Gyrodata in Proxy Video

### DISCUSSION THREAD

**Discussion by stpfl123** [gopro/labs#314] [Feature Requests]
Would be great if Proxy Video Files would contain Gyrodata to play it with ReelSteady applied in Gopro Player for very fast preview on production. 
Because many devices are not able to smoothly play 5k 10bit videos.

**Comment by dnewman-gpsw** [comment]
.LRV file do contain all the gyro data, all the same metadata as the .MP4.  I've ask the Player team if there is a workaround.

**Reply by stpfl123** [reply]
thank you for checking! 
because you said these proxies already contain gyro data i tried with gyroflow and yes it can stabilize the proxy file :-)
would be amazing if its also working with gopro player :-)
